iam.tf

```diff
@@ -279,3 +278,9 @@ resource "aws_iam_user_policy_attachment" "truefoundry_platform_user_ecr_policy_
   user       = aws_iam_user.truefoundry_platform_user[0].name
   policy_arn = aws_iam_policy.truefoundry_platform_feature_ecr_policy[0].arn
 }
+
+resource "aws_iam_user_policy_attachment" "truefoundry_platform_user_cloud_integration_policy_attachment" {
```

Comment:
This has "truefoundry_platform_user_cloud_integration_policy_attachment" instead of "truefoundry_platform_user_cloud_integration_policy_attachment"